packages/location-field/src/styled.js

```diff
@@ -138,9 +139,22 @@ export class MenuItem extends Component {
   };
 
   render() {
-    const { active, centeredText, children, disabled, header } = this.props;
+    const {
+      active,
+      centeredText,
+      children,
+      color,
+      disabled,
+      header
+    } = this.props;
     return header ? (
-      <MenuItemHeader centeredText={centeredText}>{children}</MenuItemHeader>
+      <MenuItemHeader
+        className="header"
+        bgColor={color}
```

Comment:
I would spell out `backgroundColor`.

Reply:
Do we want to restrict the customization to the background color?

Reply:
I would also prefer `backgroundColor`, but React throws errors when we do that. Have added a comment.

Reply:
Customization support added in 53460ff1daa5fa52ad45e5f503596ed50f31b8ec